drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -730,14 +753,8 @@ static int i2c_it8xxx2_transfer(const struct device *dev, struct i2c_msg *msgs,
 		data->ridx = 0;
 		data->err = 0;
 		data->msgs = &(msgs[i]);
-		data->addr_16bit = addr;
+		data->addr_16bit = addr << 1;
```

Comment:
Can you not use a 7-bit address in the driver? It seems a bit confusion to change it

Reply:
OK, removed.